src/FlaUI.TestUtilities/FlaUITestBase.cs

```diff
@@ -55,11 +55,16 @@ public abstract class FlaUITestBase
         /// </summary>
         protected virtual VideoRecordingMode VideoRecordingMode => VideoRecordingMode.OnePerTest;
 
+        /// <summary>
+        /// static member which holds the current execution date and time
+        /// </summary> 
+        private static string _testDateTime = DateTime.Now.ToString("yyyyMMddHHmmss");
+
         /// <summary>
         /// Path of the directory for the screenshots and videos for the tests.
         /// Defaults to c:\temp\testsmedia.
         /// </summary>
-        protected virtual string TestsMediaPath => @"c:\temp\testsmedia";
+        protected virtual string TestsMediaPath => $@"c:\temp\testsmedia\{SanitizeFileName(TestContext.CurrentContext.Test.Name)}\{_testDateTime}";
```

Comment:
This is kind of a breaking change but I guess most people anyway overwrite this path so we should be fine.